config/settings.yml

```diff
@@ -911,6 +911,11 @@
   :maindb: ExtManagementSystem
   :container_deployment_wizard: false
 :prototype:
+  :queue_type: miq_queue
+  :queue_hostname: localhost
+  :queue_port: 61616
+  :queue_username: admin
+  :queue_password: smartvm
```

Comment:
minor, but I think this would be better as:

```yaml
:prototype:
  :queue_type: miq_queue
  :artemis:
    :hostname: localhost
    :port: 61616
    :username: admin
    :password: smartvm
```

The way it is here, it looks like these are settings for the MiqQueue, which is just confusing.  This way it's clear these are the Artemis settings.